extensions/vertx-web/runtime/src/main/java/io/quarkus/vertx/web/runtime/VertxWebRecorder.java

```diff
@@ -87,19 +87,19 @@ public static void shutDownDevMode() {
     public static void startServerAfterFailedStart() {
         VertxConfiguration vertxConfiguration = new VertxConfiguration();
         ConfigInstantiator.handleObject(vertxConfiguration);
-        VertxRecorder.initialize(vertxConfiguration);
+        VertxRecorder.setWebVertx(VertxRecorder.initialize(vertxConfiguration));
 
         try {
             HttpConfiguration config = new HttpConfiguration();
             ConfigInstantiator.handleObject(config);
 
-            router = Router.router(VertxRecorder.getVertx());
+            router = Router.router(VertxRecorder.getWebVertx());
```

Comment:
not sure about the name as it's a bit confusing with Vert.x Web. 
should we qualified is as "RestEasy" or "internal"